website/docs/about/contribution.md

```diff
@@ -1,34 +1,200 @@
+
+## Forms of collaboration
+
+As a design system team, we are open to different forms of collaboration (even direct contribution) as long as they take the form of **joint efforts that follow the collaborative process** described below.
+
+Here is a (non exhaustive) list of possible **forms of collaboration**:
+
+- Collaborate together to build an entirely new artifact (icon, component, pattern, etc.) on the consumers’ side (details of how this collaboration will happen in practice will be defined at the project’s kickoff) with the intent that, once the artifact has been battle-tested in production, it will be brought back into the design system as an official icon/component/pattern, adapting it to follow the HDS standards and patterns.
+- Small autonomous (but coordinated) contributions like implementing bug fixes, improving existing documentation, introducing small enhancements to existing components, etc.
+- Support the product team/developers/designers on a specific task they're busy working on, to facilitate the usage of existing HDS foundations and components (or evaluate the extension of existing ones, or consider the implementation of net new ones) with the end goal of reducing their overall time-to-production.
+
+To see some examples of successful collaborations that already happened, see the [section below](#examples-of-successful-collaborations).
+
+## Collaboration process
+
+Below we outline a collaboration process that we think should fit well with the context in which the Helios Design System and the product teams operate. It’s intended as a guideline (but not a playbook) to drive the initial phase of collaboration, and provide a framework that consumer teams can follow during this collaboration.
+
+It’s not a strict model though: it’s likely that each collaboration will be handled differently, on a case-by-case basis. This nimble approach gives us space to respond to the specific needs of consumers that will arise, and better suit their specific requirements (following two of [our principles](/about/principles): _“Design in context”_ and _“Rooted in reality”_).
+
+!!! Info
+
+This list of steps is intended for large kinds of contributions/collaborations. For small kinds of contributions, we can skip certain steps, or merge multiple steps together, but the general idea would be the same: we will always work in sync with the consumers; define resources, scope and timeline; and involve all the potential stakeholders.
+
+!!!
+
+### Request
+
+The request of collaboration can come:
+
+- from a consumer team or a single product designer/developer/manager, because they think there is an opportunity of contributing to the design system or [collaborating]((#forms-of-collaboration)) with the design system team.
```

Comment:
```suggestion
- from a consumer team or a single product designer/developer/manager, because they think there is an opportunity of contributing to the design system or [collaborating]((#forms-of-collaboration)) with the design systems team.
```